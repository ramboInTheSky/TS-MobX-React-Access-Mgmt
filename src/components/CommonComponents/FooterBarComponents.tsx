import styled from '@emotion/styled'
import { variables } from '../../constants/theme'

export const ActionsBar = styled.div`
	display: flex;
	flex-direction: row;
	justify-content: space-between;
	padding: ${variables.gutter};
	/* border-bottom: ${variables.spacer}; */
	padding-bottom: calc(${variables.gutter} * 2);
	padding-top: calc(${variables.gutter} * 2);
    margin-top: calc(${variables.gutter} * 2);
	background-color: ${variables.componentBackgroundClear};
    border-top: 1px solid ${variables.componentBackground};
`
export const ButtonsContainer = styled.div`
    & button {
        margin-left: ${variables.indentation};
    }
`
