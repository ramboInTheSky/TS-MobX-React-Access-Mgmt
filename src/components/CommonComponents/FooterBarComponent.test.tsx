import React from 'react'
import { shallow } from 'enzyme'

import { ButtonsContainer, ActionsBar } from './FooterBarComponents'

describe('<FooterBarComponents />', () => {
   
    it('renders correctly ButtonsContainer', () => {
        expect(shallow(<ButtonsContainer />)).toMatchSnapshot()
		})

		it('renders correctly ActionsBar', () => {
			expect(shallow(<ActionsBar />)).toMatchSnapshot()
	})

})
