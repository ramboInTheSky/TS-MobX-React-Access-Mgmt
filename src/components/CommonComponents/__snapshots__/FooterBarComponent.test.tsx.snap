// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterBarComponents /> renders correctly ActionsBar 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;

exports[`<FooterBarComponents /> renders correctly ButtonsContainer 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
